.github/workflows/label.yml

```diff
@@ -1,19 +0,0 @@
-name: 'label'
```

Comment:
Why was this removed, just curious?

Reply:
I don't think we use it (that label doesn't even exist on some repos). I think it's an artifact from when everything was in one repo.